### PR TITLE
ts-web/core: transfer docstrings

### PR DESCRIPTION
**Pull request by pro-wh**
transferring docstrings from go codebase

also some corrections and minor additions that I noticed when I went through to do that

### CODE REVIEW

File: client-sdk/ts-web/core/src/common.ts [hunk 1]
```diff
@@ -2,27 +2,72 @@ import * as misc from './misc';
...
 export const CODE_UNKNOWN_ERROR = 1;
 
 export const IDENTITY_MODULE_NAME = 'identity';
+/**
+ * ErrCertificateRotationForbidden is returned by RotateCertificates if
+ * TLS certificate rotation is forbidden.  This happens when rotation is
+ * enabled and an existing TLS certificate was successfully loaded
+ * (or a new one was generated and persisted to disk).
+ */
 export const CODE_CERTIFICATE_ROTATION_FORBIDDEN = 1;
```
Comment (kostko) [inline client-sdk/ts-web/core/src/common.ts:71]: The identity module is not exposed to clients so it probably doesn't make much sense to transfer it over?

**Review by kostko — APPROVED** [review]
I see this is based on 20.12? Can we update it for latest master (in a separate PR) where some things are quite different?

File: client-sdk/ts-web/core/src/common.ts (hunk 1 above)
Comment (pro-wh) [reply client-sdk/ts-web/core/src/common.ts:71]: oh I had searched for all calls to `errors.New`. let me open an issue for tracking this and related removals

Comment (pro-wh) [reply client-sdk/ts-web/core/src/common.ts:71]: #8

**Comment by pro-wh** [comment]
> I see this is based on 20.12? Can we update it for latest master (in a separate PR) where some things are quite different?

yup. I've opened #9 to track this

**Pull request merged**